Source/WebCore/platform/cocoa/ContentFilterUnblockHandlerCocoa.mm

```diff
@@ -118,7 +119,8 @@
     unblockHandler.m_unblockURLHost = [coder decodeObjectOfClass:[NSString class] forKey:unblockURLHostKey];
     unblockHandler.m_unreachableURL = [coder decodeObjectOfClass:[NSURL class] forKey:unreachableURLKey];
 #if HAVE(PARENTAL_CONTROLS_WITH_UNBLOCK_HANDLER)
-    unblockHandler.m_webFilterEvaluator = [coder decodeObjectOfClass:getWebFilterEvaluatorClass() forKey:webFilterEvaluatorKey];
+    if (WebContentFilterLibrary())
+        unblockHandler.m_webFilterEvaluator = [coder decodeObjectOfClass:webFilterEvaluatorClass forKey:webFilterEvaluatorKey];
```

Comment:
Shouldn't this still be `getWebFilterEvaluatorClass()`? I don't see where `webFilterEvaluatorClass` is defined.

Reply:
> Shouldn't this still be `getWebFilterEvaluatorClass()`? I don't see where `webFilterEvaluatorClass` is defined.

🤦 yes (I originally had `auto webFilterEvaluatorClass  = getWebFilterEvaluatorClass()`, changed it to `WebContentFilterLibrary()` and forgot to change the `webFilterEvaluatorClass`)

Reply:
> > Shouldn't this still be `getWebFilterEvaluatorClass()`? I don't see where `webFilterEvaluatorClass` is defined.
> 
> 🤦 yes (I originally had `auto webFilterEvaluatorClass = getWebFilterEvaluatorClass()`, changed it to `WebContentFilterLibrary()` and forgot to change the `webFilterEvaluatorClass`)

I undid all of my changes to this file since it's all `PLATFORM(IOS_FAMILY)` only.